_data/fa-owners.yml

```diff
@@ -0,0 +1,353 @@
+'API definition':
+  - area: 'API definition'
+    label: api-definition
+    owner:
+    - sq-lb-epitome
+    repos:
+    - loopback-api-definition
+    - loopback-component-explorer
+    - loopback-swagger
+
+'Boot':
+  - area: Boot
+    label: boot
+    owner:
+    - sq-lb-asteroid
+    repos:
+    - loopback-boot
+    - loopback-example-app-logic
+    smes:
+    - davidcheung
```

Comment:
In the table as displayed, all of these are rendered as links, GitHub IDs, repos, and teams.  The links are added in https://github.com/strongloop/loopback.io/pull/216/files#diff-fab19d40a489a9c5ba42e6dfe95b57b2

If you run the site yourself, you'll see how it works, but you can get an idea from the screenshot I posted...